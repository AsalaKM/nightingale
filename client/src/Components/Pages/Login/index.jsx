
import React from 'react';

export default function index () {
  return (
    <div >
      <p>Login.</p>
    </div>
  );
}