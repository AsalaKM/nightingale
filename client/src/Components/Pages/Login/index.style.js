import styled from "styled-components";

export const LoginWrapper = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
  width: 100%;
  padding: 50px 0;
`;

export const AppTitle = styled.img`
  display: block;
  object-fit: contain;
  width: 80%;
  max-width: 400px;
`;

export const AppLogo = styled.img`
  display: block;
  object-fit: contain;
  width: 50%;
  max-width: 250px;
`;

export const Form = styled.form`
  margin: 0 auto;
  display: flex;
  flex-direction: column;
  align-item: center;
  width: 85%;
  padding: 25px 0;
  box-shadow: 0px 0px 6px #ccc;
  margin-top: 25px;
  border-radius: 8px;
  max-width: 500px;

  input {
    width: 100%;
    max-width: 400px;
    margin: 0 auto;
  }

  button {
    max-width: 100px;
    margin: 12px auto;
  }

  h1 {
    border: none;
  }
`;
