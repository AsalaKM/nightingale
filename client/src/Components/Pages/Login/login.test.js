import React from "react";
import renderer from "react-test-renderer";
import { StaticRouter } from "react-router-dom";

import Login from "./index.jsx";

describe("Login must match the snapshots", () => {
  it("renders correctly", () => {
    const context = {};
    const tree = renderer
      .create(
        <StaticRouter context={context}>
          <Login />
        </StaticRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
