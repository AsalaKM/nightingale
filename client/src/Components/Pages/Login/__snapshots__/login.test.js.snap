// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login must match the snapshots renders correctly 1`] = `
<div
  className="sc-EHOje kHDAue"
>
  <img
    className="sc-bZQynM iCtEDl"
    src="header.png"
  />
  <img
    className="sc-gzVnrw hPXWJT"
    src="logo.png"
  />
  <form
    className="sc-htoDjs QlDZz"
    onSubmit={[Function]}
  >
    <div
      className="sc-bdVaJa khUSez"
    >
      <h1
        className="sc-bwzfXH dtARpq"
      >
        Login
      </h1>
    </div>
    <label
      className="sc-htpNat duRZTP"
      htmlFor="username"
    >
       
      <input
        className="sc-bxivhb iHLnwu"
        name="username"
        onChange={[Function]}
        placeholder="Username"
        style={
          Object {
            "width": undefined,
          }
        }
        value=""
      />
    </label>
    <label
      className="sc-htpNat duRZTP"
      htmlFor="password"
    >
       
      <input
        className="sc-bxivhb iHLnwu"
        name="password"
        onChange={[Function]}
        placeholder="Password"
        style={
          Object {
            "width": undefined,
          }
        }
        type="password"
        value=""
      />
    </label>
    <button
      className="sc-ifAKCX iAgQtD"
      style={
        Object {
          "background": undefined,
        }
      }
      type="submit"
    >
      Login
    </button>
    <p
      className="sc-dnqmqq fXmtAI"
    >
      don't have an account?
       
      <a
        className="sc-iwsKbI hTGocj"
        href="/signup"
        onClick={[Function]}
      >
        sign up now!
      </a>
    </p>
  </form>
</div>
`;
